### PR TITLE
added installation comment about required packages for county choropleth

### PR DESCRIPTION
**Pull request by Kully**
https://github.com/plotly/documentation/issues/959

cc @cldougl ready for review

### CODE REVIEW

File: _posts/python/maps/county-choropleth/2015-06-30-county_choropleth.html [hunk 1]
```diff
@@ -59,14 +59,32 @@ <h4 id="Version-Check">Version Check<a class="anchor-link" href="#Version-Check"
 
 
 <div class="output_text output_subarea output_execute_result">
-<pre>&#39;2.5.1&#39;</pre>
+<pre>&#39;3.0.0rc9&#39;</pre>
```
Comment (cldougl) [inline _posts/python/maps/county-choropleth/2015-06-30-county_choropleth.html]: @Kully can we keep the version check on version 2.x.x for now?
I think we should we should move the version check to 3.x.x for all docs simultaneously when it is released as stable.

File: _posts/python/maps/county-choropleth/2015-06-30-county_choropleth.html [hunk 2]
```diff
@@ -59,14 +59,32 @@ <h4 id="Version-Check">Version Check<a class="anchor-link" href="#Version-Check"
...
 
 </div>
 </div>
 
+</div>
+<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
+</div>
+<div class="inner_cell">
+<div class="text_cell_render border-box-sizing rendered_html">
+<h4 id="Install-these-Packages">Install these Packages<a class="anchor-link" href="#Install-these-Packages">&#194;&#182;</a></h4><p><code>geopandas</code>, <code>pyshp</code> and <code>shapely</code> must be installed for this figure factory.</p>
```
Comment (cldougl) [inline _posts/python/maps/county-choropleth/2015-06-30-county_choropleth.html]: Let's title this section Additional Packages or Required Packages

File: _posts/python/maps/county-choropleth/2015-06-30-county_choropleth.html [hunk 3]
```diff
@@ -59,14 +59,32 @@ <h4 id="Version-Check">Version Check<a class="anchor-link" href="#Version-Check"
...
+<p>Run the following commands to install the correct versions of the following modules:</p>
```
Comment (cldougl) [inline _posts/python/maps/county-choropleth/2015-06-30-county_choropleth.html]: And explain: The following packages are required for this figure factory. Run the following commands to install the correct versions of the following modules:

**Comment by cldougl** [comment]
@michaelbabyn would you be able to pick up this pr from here and apply the requested changes?

**Comment by michaelbabyn** [comment]
@cldougl I made all of the changes and it looks good. I also added a jupyter notebook link

**Pull request merged**